etcdmain/etcd.go

```diff
@@ -527,6 +527,24 @@ func startProxy(cfg *config) error {
 		}
 	}
 
+	if len(cfg.ec.CipherSuites) == 0 {
+		if lg != nil {
+			lg.Info("Using default cipher suites")
+		} else {
+			plog.Info("Using default cipher suites")
+		}
+	} else {
+		if lg != nil {
+			lg.Info("Using cipher suites:", zap.Strings("cipher-suites", cfg.ec.CipherSuites))
```

Comment:
@rkday besides fixing a fmt error as in build, I would suggest to use lower case for all the log messages (using vs Using ..) per convention. Thanks!